### PR TITLE
Add netdata

### PR DESCRIPTION
**Pull request by cakrit**
Add netdata based on the helm chart at https://github.com/netdata/helmchart

Note the following in `deploy-local.sh` and `deploy.sh`, not 100% sure that this is what we need, but we don't use a deployment.

```
# ensure services are running
kubectl rollout status statefulset/netdata-master
kubectl rollout status daemonset/netdata-slave
```

### CODE REVIEW

File: stacks/netdata/render.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+# Create YAML directory
+rm -rf "$ROOT_DIR"/stacks/netdata/yaml
+mkdir -p "$ROOT_DIR"/stacks/netdata/yaml
+
+# render mginx-ingress
+helm template \
+  --name netdata \
+  --namespace netdata \
+  --values $SRC_VERSION/values.yaml \
```
Comment (countingtoten) [inline stacks/netdata/render.sh]:
```suggestion
  --values $SRC_VERSION/values.yaml \
  --set master.database.storageclass=null \
  --set master.alarms.storageclass=null \
  --set master.alarms.volumesize=1Gi \
```

**Comment by countingtoten** [comment]
@cakrit, thank you for the submission! 

When I used deploy-local.sh to deploy to my Kubernetes cluster, the it failed to create the PersistentVolumeClaims for two reasons.
1) DigitalOcean currently only supports `do-block-storage` as a storage class. If you leave the `storageClassName` out of your yaml, kubectl will use the default storage class. This is `do-block-storage` in our case.
2) The minimum size for a DigitalOcean PVC is 1 gigabyte. We increase the size of the alarms PVC.

I included these changes as a Github suggestion. Let me know if these changes work for you, and we can continue the release process!

Thanks,
Jim

**Comment by countingtoten** [comment]
Another option would be to have a values.yaml in the stack/netdata directory that overwrites the `master.database` and `master.alarms` fields.

**Comment by countingtoten** [comment]
I deleted my Github suggestion. Our prometheus stack uses a values.yaml file that overwrites the Helm template values.yaml. I think I like that better.

**Comment by cakrit** [comment]
Thanks, I'll change the `helm template` command in `render.sh` to overwrite the `values.yaml` from `src`, with a second `values.yaml` under `stacks`. I just need to verify that helm will properly overwrite the settings in the first file with the settings in the second.

**Comment by cakrit** [comment]
Worked well, I disabled the alarm log persistence. Not that important, so I didn't want to add another 1GiB just for that.

**Pull request merged**